content/about/notes/1.1/index.md

```diff
@@ -18,15 +18,50 @@ TODO: Need to have an explicit section on everything that's being deprecated.
 
 ## Upgrades
 
-- **Helm Changes**.
-TBD: need full content & links
-
-- things to watch out for/configure before you upgrade - from all subsystems below. For example, outbound traffic is allowed by default for unknown ports, what else?
-TBD: need full content & links
-
-    - Egress gateway is disabled by default
-    - Mixer policy is off by default
-    - Outbound traffic policy is set to `ALLOW_ANY`. Traffic to unknown ports will be forwarded as-is.
+- **New `istio-init` Helm Chart**. Istio’s CRDs have been placed into their own Helm chart `istio-init`.
```

Comment:
I am very wary of adding all these details in the updates section. I think it is better to provide a link to it.

Reply:
ddo you mean all of upgrade, or simiply iistio-init helm chart?  @geeknoid had suggested an "Upgrade notes" section.  I think we want to be as crystal clear as possible that if folks don't install istio-init, bad things happen (or install the crds).

Reply:
all of the upgrade... as it is a huge daunting list in our release note.

Reply:
We could rename call the ``## Upgrades` line on 19 to `## Installation` and move the upgrade notes off into an upgrade section as per @geeknoid 's suggestion.

Reply:
all we need in release note is we need to ensure user visit the upgrade link (that has all these upgrade items and the upgrade doc before performing update.)